core/dbt/task/run.py

```diff
@@ -105,9 +106,9 @@ def track_model_run(index, num_nodes, run_model_result):
         "index": index,
         "total": num_nodes,
         "execution_time": run_model_result.execution_time,
-        "run_status": run_model_result.status,
-        "run_skipped": run_model_result.skip,
-        "run_error": None,
+        "run_status": str(run_model_result.status).upper(),
+        "run_skipped": run_model_result.status == NodeStatus.Skipped,
+        "run_error": run_model_result.status == NodeStatus.Error,
```

Comment:
This will affect model run event tracking data, I'm not sure if there is anything else to consider here.

Reply:
thanks for calling it out! we'll just need to account for this on our end